### ISSUE TITLE
Fix inner scroll on for-you page

### ISSUE THREAD

**Issue by CowMuon** [hicommonwealth/commonwealth#2871]
Shouldn't need a loom here, this is fairly obvious when you're on the page.

**Comment by dillchen** [comment]
I am not sure this behavior is desireable. That is current behavior may be suitable.

When we look at Twitter, only the feed is scrollable, in this case, our feed is scrollable. In both cases all other elements on the page are sticky (I.e Communities to Explore remain sticky)

**Comment by zakhap** [comment]
Not exactly right, re: twitter. The header and sidebars are _floating_ but the scroll is still at the high level of the page. See right scroll bar in image below. This means you can be anywhere on the page _that does not have an inner scroll_ and scroll the main body. Having the for-you page be an inner scroll means it scrolls like twitter dm message list. 


![Image](https://user-images.githubusercontent.com/16248081/226481237-19c4fe71-36b9-4195-b620-9c8d0e74859e.png)

**Comment by dillchen** [comment]
Great. this is better defined now. I would suggest adding an additional label to `React-a-thon` for tickets that may potentially not be blocking on getting react on production. 

Of which this may be one. And strictly prioritizing things that we consider blocking

**Comment by zakhap** [comment]
Definitely! However, I still think this should be addressed as part of the react migration as it gives us a standard to follow for other paginated/infinite-scroll pages (notifications page, thread listing page, community feed, etc). It's not particularly heavy but it's impactful 💪

**Comment by CowMuon** [comment]
Can we get a status update here please? This does not seem to be a 1 pointer based just on how it has been going so far.

**Comment by masvelio** [comment]
@CowMuon this needs some CSS adjustments in https://github.com/hicommonwealth/commonwealth/pull/3251 and will be good to go
